### ISSUE TITLE
foggy cityscape dataset result

### ISSUE THREAD

**Issue by ddghost** [Megvii-Nanjing/CR-DA-Det#6]
Hi, 

I follow the README steps to train the network, but the best MAP is only 35. I wonder if my train_s.txt, train_t.txt and test_t.txt differ from yours. Could you provide me these three files?

**Comment by verai11** [comment]
I wonder how do you get "cityscape_2007_train_s","cityscape_2007_train_t",...from cityscape dataset,could you share it with me?thank you!!

**Comment by ddghost** [comment]
> I wonder how do you get "cityscape_2007_train_s","cityscape_2007_train_t",...from cityscape dataset,could you share it with me?thank you!!

Hi, you can follow the instruction of https://github.com/VisionLearningGroup/DA_Detection or https://github.com/yuhuayc/da-faster-rcnn

**Comment by verai11** [comment]
Are you talking about those matlab files?I did notice that,too.That means I must install MATLAB?what' more,the data format generated by them is consistent with what I need or need some adjustments?

**Comment by ddghost** [comment]
> Are you talking about those matlab files?I did notice that,too.That means I must install MATLAB?what' more,the data format generated by them is consistent with what I need or need some adjustments?

Yes, you need to install MATLAB, but I have already forgot the details.

**Comment by verai11** [comment]
Got it.Thank you!

**Comment by verai11** [comment]
Hi,Sorry to bother you again.How long did it take you to train this model?What type of GPU do you have?

**Comment by ddghost** [comment]
> Hi,Sorry to bother you again.How long did it take you to train this model?What type of GPU do you have?

Training 100 iterations takes about one minute by TITAN X.
